src/code/game.c

```diff
@@ -1,9 +1,20 @@
 #include "global.h"
+#include "sys_cfb.h"
 #include "system_malloc.h"
 #include "z64debug_text.h"
 #include "z64rumble.h"
+#include "z64vimode.h"
+#include "z64viscvg.h"
+#include "z64vismono.h"
+#include "z64viszbuf.h"
 #include "overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h"
 
+extern UNK_TYPE1 D_801F7FF0;
+extern VisCvg sVisCvg;
+extern VisZbuf sVisZbuf;
+extern VisMono sMonoColors;
```

Comment:
```suggestion
extern VisMono sVisMono;
```
Need to change the one in Play, but I think this one is better as `sVisMono` to match the style of the others.